### ISSUE TITLE
insert Return to ID

### ISSUE THREAD

**Issue by phpjishu** [erikolson186/zangodb#25]
Insert should return to ID

**Comment by aFelipeSP** [comment]
this is really important, how can I get the id of the inserted item?

**Comment by braadworst** [comment]
Yes this would be a nice feature to have. Thanks for buildingn this lib, really enjoying it!
